setup.py

```diff
@@ -30,23 +30,25 @@
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python",
         "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.6",
         "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
         "Topic :: Software Development :: Quality Assurance"
     ],
     keywords=[
         "twisted", "checker", "compliance", "pep8"
     ],
     install_requires=[
-        "pylint == 0.26.0",
-        "logilab-common == 0.62.0",
-        "pep8 == 1.5.7"
+        "pylint == 1.5.6",
+        "logilab-common == 1.2.1",
```

Comment:
This is extra-confusing then. Aren't we using it to import the exception types and such required by pylint?  If pylint doesn't use them any more, can we just delete that code?